### Same "cl:functions:two-way-stream-input-stream"
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:two-way-stream-output-stream.txt	2017-04-27 21:42:55.000000000 +0100
+++ saved-pages-clus/cl:functions:two-way-stream-output-stream.txt	2017-04-27 21:42:55.000000000 +0100
@@ -3,9 +4,9 @@
 Syntax


-two-way-stream-input-stream two-way-stream => input-stream
+ two-way-stream-input-stream  two-way-stream → input-stream
+ two-way-stream-output-stream two-way-stream → output-stream

-two-way-stream-output-stream two-way-stream => output-stream

 Arguments and Values

@@ -9,21 +12,58 @@

 Arguments and Values

 two-way-stream - a two-way stream.
 input-stream - an input stream.
 output-stream - an output stream.
 Description

 two-way-stream-input-stream returns the stream from which two-way-stream receives input.
 two-way-stream-output-stream returns the stream to which two-way-stream sends output.
-Examples None.

-Side Effects None.

-Affected By None.
+Examples
+None.
+Side Effects
+None.
+Affected By
+None.
+Exceptional Situations
+None.
+See Also
+None.
+Notes

-Exceptional Situations None.
+None.

-See Also None.

-Notes None.
\ No newline at end of file
+\
\ No newline at end of file
